scripts/bump-version.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ "$(uname -s)" != "Linux" ]; then
+    echo "This script can only be run on Linux!"
+    echo "Please use the GitHub Action instead."
+    exit 1
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $SCRIPT_DIR/..
+
+OLD_VERSION="${1}"
+NEW_VERSION="${2}"
+
+echo "Current version: ${OLD_VERSION}"
+echo "Bumping version: ${NEW_VERSION}"
```

Comment:
Does craft read the output of this script?

Reply:
Nope. Craft only need this script to run, so the changelog will be properly generated.